lib/wovnrb/services/html_converter.rb

```diff
@@ -21,6 +21,10 @@ def build_api_compatible_html
     private
 
     def html
+      # Ensure a Content-Type declaration in the header. This mimics Nokogumbo
+      # 1.5.0 default serialization behavior.
+      @dom.meta_encoding = 'UTF-8' if @dom.respond_to?(:meta_encoding=)
```

Comment:
Do we ever expect to not want to output UTF-8? Should we be detecting this encoding from the user's system?

Reply:
traditionally we have always outputted UTF-8, so i think it's fine. Because target languages on their project may not be supported by whatever the user's original encoding was.